Assets/MixedRealityToolkit-SDK/Profiles/DefaultMixedRealityBoundaryVisualizationProfile.asset

```diff
@@ -15,14 +15,19 @@ MonoBehaviour:
   boundaryHeight: 3
   showFloor: 1
   floorMaterial: {fileID: 2100000, guid: 47f3c5e1cb6142ba9697cd4c86d74321, type: 2}
+  floorPhysicsLayer: 2
```

Comment:
```suggestion
  floorPhysicsLayer: 0
```

Reply:
just noticed that I forgot to tag the .asset file with the floor layer fix. sorry about that